libcontainer/standard_init_linux.go

```diff
@@ -86,15 +86,15 @@ func (l *linuxStandardInit) Init() error {
 	// initialises the labeling system
 	selinux.GetEnabled()
 
-	// We don't need the mountFds after prepareRootfs() nor if it fails.
+	// We don't need the mount nor idmap fds after prepareRootfs() nor if it fails.
 	err := prepareRootfs(l.pipe, l.config, l.mountFds)
-	for _, m := range l.mountFds {
+	for _, m := range append(l.mountFds.sourceFds, l.mountFds.idmapFds...) {
 		if m == -1 {
 			continue
 		}
 
 		if err := unix.Close(m); err != nil {
-			return fmt.Errorf("Unable to close mountFds fds: %w", err)
+			return fmt.Errorf("unable to close idmapFds and sourceFds: %w", err)
```

Comment:
I think maybe we should keep the description of `mountFds fds`, otherwise it will be confused for someone who doesn't use idmap mounts.
Or change to close `sourceFds` and `idmapFds` seperatelly?

Reply:
@lifubang  Right, makes sense! I've updated it to keep the current description. Thanks!